packages/sdk/src/swap/types.ts

```diff
@@ -59,6 +59,7 @@ export interface CommonStatusFields extends ChainsAndAssets {
   expectedDepositAmount: string | undefined;
   depositChannelExpiryBlock: bigint;
   estimatedDepositChannelExpiryTime: number | undefined;
+  isExpired: boolean;
```

Comment:
as this is the status response, i think we need to be explicit that this is about the deposit channel
```suggestion
  depositChanneIsExpired: boolean;
```